src/index.html

```diff
@@ -7,8 +7,42 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <span class="logo"><img src="images/logo.png" alt=""></span>
```

Comment:
```suggestion
      <span class="logo">
        <img src="images/logo.png" alt="">
      </span>
```
Один тег - одна линия

Reply:
```suggestion
      <span class="logo"><img src="images/logo.png" alt="logo"></span>
```
Не оставляй alt пустым